prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -104,6 +105,15 @@ if [[ ! -z $JIGASI_XMPP_PASSWORD ]]; then
     prosodyctl --config $PROSODY_CFG register $JIGASI_XMPP_USER $XMPP_AUTH_DOMAIN $JIGASI_XMPP_PASSWORD
 fi
 
+if [[ ! -z $JIGASI_TRANSCRIBER_PASSWORD ]]; then
+    OLD_JIGASI_TRANSCRIBER_PASSWORD=passw0rd
```

Comment:
This never existed, so we don't need to check for it.

Reply:
Honestly I just copied the handling of other paswords. I don't know why this password would have to be handled specially. Or is this some sort of legacy stuff that is simply no longer necessary for newly introduced passwords?

Reply:
The latter.